src/services/auth.services.ts

```diff
@@ -0,0 +1,29 @@
+import Axios from 'axios'
+import { addAxiosInterceptors } from 'sbc-common-components/src/util/interceptors'
+
+const axios = addAxiosInterceptors(Axios.create())
+
+export default class AuthServices {
+  /** The Auth API URL. */
+  static authApiUrl = sessionStorage.getItem('AUTH_API_URL')
+
+  /** Fetches current user's info. */
+  static async fetchUserInfo (): Promise<any> {
+    const url = `${this.authApiUrl}users/@me`
+    return axios.get(url)
+      .then(response => {
+        if (response?.data) return response.data
+        throw new Error('Invalid user info')
+      })
+  }
+
+  /** Fetches specified org's info. */
+  static async fetchOrgInfo (orgId: number): Promise<any> {
+    const url = `${this.authApiUrl}orgs/${orgId}`
+    return axios.get(url)
+      .then(response => {
+        if (response?.data) return response.data
+        throw new Error('Invalid org info')
+      })
+  }
+}
```

Comment:
Do you think we should move legal-api-mixin.ts in this services folder too? Is it as easy as moving the file and a bit of renaming?

Reply:
Sorry, i didn't realize we already captured these requests in a mixin so i removed my additional code.

Reply:
I just noticed that too. OK, nvm :)